### PR TITLE
Downgrade php requirements

### PR DESCRIPTION
**Pull request by mickaelandrieu**
Hello,

I have no previous experience with AppVeyor, so I've made a second commit so I can have help on this specific subject :)

Mickaël

### CODE REVIEW

**Comment by mickaelandrieu** [comment]
ping @dunglas, wdyt? What should I do to fix the PHPStan issue?

File: composer.json [hunk 1]
```diff
@@ -17,10 +17,11 @@
     }
   ],
   "require": {
-    "php": ">=7.2",
+    "php": ">=7.1",
     "facebook/webdriver": "^1.5",
     "symfony/browser-kit": "^4.0",
-    "symfony/process": "^4.0"
+    "symfony/process": "^4.0",
+    "symfony/polyfill-php72": "^1.9"
```
Comment (xabbuh) [inline composer.json]: should be before `symfony/process` (Composer would resort the packages the next time anyway thanks to the settings below)

File: appveyor.yml [hunk 2]
```diff
@@ -5,11 +5,17 @@ clone_folder: c:\projects\symfony\panther
 cache:
```
[the remark this answers is not in the document]
Comment (mickaelandrieu) [reply appveyor.yml]: ok, I'll update asap

**Comment by dunglas** [comment]
Thanks @mickaelandrieu

**Pull request merged**